apps/web/app/[orgShortCode]/settings/SettingsSidebar.tsx

```diff
@@ -1,11 +1,210 @@
 'use client';
 
-import { Flex } from '@radix-ui/themes';
+import React from 'react';
```

Comment:
just a few nitpicks,

```suggestion
import { type ReactNode } from 'react';
```